core/domain/role_services.py

```diff
@@ -158,6 +164,7 @@
         ACTION_ACCESS_CREATOR_DASHBOARD,
         ACTION_ACCESS_LEARNER_DASHBOARD,
         ACTION_CREATE_EXPLORATION,
+        ACTION_CREATE_NEW_QUESTION,
```

Comment:
Nope, this is topic manager. Exploration editors can _suggest_ questions, but that comes later. Suggesting a possible addition is different from actually creating.

Reply:
Done, removed this, since the decorator that used this was not called anywhere.